product_docs/docs/net_connector/8.0.5.1/release_notes/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Release notes"
+navigation:
+- 8.0.5.1_rel_notes
+- 8.0.2.1_rel_notes
+- 7.0.6.2_rel_notes
+- 7.0.6.1_rel_notes
+- 7.0.4.1_rel_notes
+- 6.0.2.1_rel_notes
+- 5.0.7.1_rel_notes
+- 4.1.6.1_rel_notes
+- 4.1.5.1_rel_notes
+- 4.1.3.1_rel_notes
+- 4.0.10.2_rel_notes
+- 4.0.10.1_rel_notes
+- 4.0.6.1_rel_notes
+---
+
+The EDB .NET connector documentation describes the latest version of EDB .NET connector. 
+
+These release notes describe what's new in each release. When a minor or patch release introduces new functionality, indicators in the content identify the version that introduced the new feature.
+
+| Version                                  | Release Date | 
+| ---------------------------------------- | ------------ | 
+| [8.0.5.1](8.0.5.1_rel_notes)  | 22 Nov 2024      |
+| [8.0.2.1](8.0.2.1_rel_notes)  | 15 May 2024      |
+| [7.0.6.2](7.0.6.2_rel_notes)  | 15 Feb 2024      |
+| [7.0.6.1](7.0.6.1_rel_notes)  | 25 Oct 2023      |
+| [7.0.4.1](7.0.4.1_rel_notes)  |       |
```

Comment:
@xfischer  I could not figure out the release dates on this version and previous versions. Is there a place in your repo where I can check all release dates? Thank you!

Reply:
Hi Gaby, there are tags created for each release, here's what I have
| 7.0.4.1  | 07 Jul 2023 |
| 6.0.2.1  | 25 Jul 2022 |
| 5.0.7.1  | 06 Aug 2021 |
| 4.1.6.1  | 17 Dec 2020  |
| 4.1.5.1  | 11 Nov 2020  |
| 4.1.3.1  | 27 Aug 2020 |
| 4.0.10.2 | 12 Mar 2020 |
| 4.0.10.1 | 26 Sep 2019 |